src/plugins/hotkeys/README.md

```diff
@@ -0,0 +1,53 @@
+# Plugin: Hotkeys
+
+Supports for hotkeys to add shortcuts.
+
+## Overview
+
+| <!-- -->         | <!-- -->                                     |
+| ---------------- | -------------------------------------------- |
+| Name             | `pip`                                        |
+| Path             | `vlitejs/plugins/hotkeys`                 |
+| Entry point      | `vlitejs/plugins/hotkeys/hotkeys.js`   |
+| Provider&sup2;   | `'html5', 'youtube', 'vimeo', 'dailymotion'` |
+| Media type&sup3; | `'video', 'audio'`                           |
+
+## Hotkeys
+
+The player accepts the following keyboard shortcuts.
+
+|        Key        | Action                   |
+| :---------------: | ------------------------ |
+| <kbd>space</kbd>  | Toggle playback          |
+|  <kbd>Esc</kbd>   | Exit the fullscreen      |
+| <kbd>&larr;</kbd> | Seek backward of `5s`    |
+| <kbd>&rarr;</kbd> | Seek forward of `5s`     |
+| <kbd>&uarr;</kbd> | Increase volume of `10%` |
+| <kbd>&darr;</kbd> | Decrease volume of `10%` |
+
+## Usage
+
+### HTML
+
+```html
+<video id="player" src="<path_to_video_mp4>"></video>
+```
+
+### JavaScript
+
+```js
+import 'vlitejs/vlite.css';
+import Vlitejs from 'vlitejs';
+import VlitejsHotkeys from 'vlitejs/plugins/hotkeys.js';
+
+Vlitejs.registerPlugin('hotkeys', VlitejsHotkeys);
+
+new Vlitejs('#player', {
+  plugins: ['hotkeys']
+});
+```
+
+## Demo
+
+# TODO
+See the [volume-bar plugin on audio](https://glitch.com/edit/#!/vlitejs-html5-audio-volume-bar?previewSize=50&attributionHidden=false&sidebarCollapsed=false&path=index.html&previewFirst=false) and [volume-bar plugin on video](https://glitch.com/edit/#!/vlitejs-html5-video-volume-bar?previewSize=50&attributionHidden=false&sidebarCollapsed=false&path=index.html&previewFirst=false) demo.
```

Comment:
Not sure about this section. I'm not fan of the Glitch website for demos. You can remove this. Later demos will probably be back in the project and hosted in Vercel